node/src/main/protobuf/waves/database.proto

```diff
@@ -34,7 +40,7 @@ message TransactionMeta {
   int32 height = 1;
   int32 num = 2;
   int32 type = 3;
-  bool failed = 4;
+  Status status = 4;
```

Comment:
Interesting. Can we have here some kind of backward compatibility so probably we don't need to rebuild a state? I mean:
1. Can we encode `FAILED` as 1
```scala
  @SerialVersionUID(0L)
  case object FAILED extends Status(0) with Status.Recognized {
    val index = 0
```
`SUCCEEDED` as 0 and `ELIDED` as 2?
2. Will we have same bytes in protobuf for `FAILED` and `SUCCEEDED` as it was `bool failed` (I hope so)?

Reply:
This changes appears only in 1.5 version, which anyway requires state rebuilding at least due to a change in storage